content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-certs.md

```diff
@@ -123,8 +121,6 @@ These are advanced topics for users who need to integrate their organization's c
 
 The Kubernetes Certificate Authority does not work out of the box.
 You can configure an external signer such as [cert-manager][cert-manager-issuer], or you can use the built-in signer.
-The built-in signer is part of [`kube-controller-manager`][kcm].
-To activate the built-in signer, you pass the `--cluster-signing-cert-file` and `--cluster-signing-key-file` arguments.
```

Comment:
nit: This wording (without “must”) seems better than the other one. I would keep these lines and omit the ones below.

Reply:
@PabloCastellano could you please incorporate the changes that @sftim suggested.

Reply:
done